### ISSUE TITLE
Add jobs

### ISSUE THREAD

**Issue by Seraph2** [OpenCityGaming/citymod#13]
Proposed job categories:
Citizen, Criminal, Government

**Comment by livkx** [comment]
- President
- Vice (maybe)
- Swat Chief
- Swat
- Police Chief
- Police Officer
- Medic
- Fireman
- Chef
- Repairman / Handyman
- Mechanic
- Citizen
- BMD
- Gun Dealer
- Thug
- Mafia Boss
- Mafia Member

**Comment by Seraph2** [comment]
I propose "Mayor" instead of "President.

Fireman might need some time before it should (potentially) be added.

Mechanic and repairman will need some tasks to them before they can be added.

BMD may need to be revised if it could potentially fit another job imo.

How come thug has no leader?

I feel mafia is okay, or maybe "rebel" or so. "Criminal" could potentially work as well.

**Comment by livkx** [comment]
Agreed with all points except mafia should stay mafia, thug is in place to be the lesser version of mafia

thx

**Comment by Seraph2** [comment]
Alright, makes sense.
